wolfSSL/src/chapter02.md

```diff
@@ -394,6 +394,122 @@ Removes code for OAEP padding.
 
 Turns off AES-CBC algorithm support.
 
+#### WOLFCRYPT_ONLY
+
+Enables wolfCrypt only while disabling TLS.
+
+#### NO_CAMELLIA_CBC
+
+Disables Camellia CBC support.
+
+#### NO_AES
+
+Disables AES algorithm support.
+
+#### NO_AESGCM_AEAD
+
+Disables AES GCM algorithm support.
+
+#### NO_ASN_TIME
+
+Disables time checking for ASN.
+
+#### NO_CHECK_PRIVATE_KEY
+
+RSA - needed to encrypt salt.
+
+#### NO_DH
+
+Disables DH support.
+
+#### NO_ED25519_CLIENT_AUTH
+
+Disables client authEd25519 requires caching enabled for tracking message hash used in EdDSA_Update for signing.
+
+#### NO_ED448_CLIENT_AUTH
+
+Disables client authentification for ED448.
+
+#### NO_FORCE_SCR_SAME_SUITE
+
+Applys a restriction on secure negotation. Don't allocate Suites' object on renegotiation.
```

Comment:
By default secure renegotiation requires using the same cipher suite. This disables that requirement.